logger/log.go

```diff
@@ -0,0 +1,33 @@
+package logger
```

Comment:
We will have a new logger interface in future to allow customer to inject io.writer. What is more, we will also support logger per instance instead of just support global logger.